services/asset/nodes/schemas.go

```diff
@@ -157,15 +211,6 @@ var dataSourceFilterSchema = map[string]*schema.Schema{
 			Description:  helper.AllowedValuesToDescription(validNodeTypes),
 		},
 	},
-	"kinds": {
```

Comment:
it shouldn't be removed, right?

Reply:
Right. It hasn't been removed. You can see it between `parent_node_ids` and `is_root_node`.

Reply:
right, I didn't see it :/